meetings/membership-criteria.md

```diff
@@ -0,0 +1,64 @@
+
+## Basic information
+
+First, we validate the personal information:
+
+- Reasonable first name, last name.
+- valid email address: We only validate that it is a valid email address format. Later, when an invitation is sent, that verifies that it's a real address.
+- valid GitHub ID. We validate the profile appears to be human. There should be some activity. The user name must match any contribution links provided. GitHub activity mentioned in the contribution statement should match this ID.
+
+## Open Source Maintainer, Open Source Contributor
+
+See [Open source maintainer](membership-participation-types.md#the-open-source-maintainer) and the [Open source contributor](membership-participation-types.md#the-open-source-contributor).
+
+We look at the contribution links. All activity must conform to the .NET Foundation CoC. Activities examined include:
+
+- PRs created. We evaluate the participation in the community. Does the PR match the project's guidelines? Does the author engage constructively with reviewers?
+- PR review, and helping new contributors. Are comments in line with the spirit of the CoC? Does the reviewer provide necessary assistance?
+- Issues created. These should be well-defined issues. We don't evaluate issue quality based on the project maintainer's decision to accept or reject the idea. We do evaluate issues based on the clarity and completeness.
+- Discussion forums. Does the person participate in discussions in a constructive manner?
+
+A person need not participate in all these areas, but should be active in some of them.
+
+## Event organizer
+
+See [Event organizer particpation type](membership-participation-types.md#the-event-organizer).
+
+We look at the event or user group activities. Examples include:
+
+- Events listed have a CoC that reflects the .NET Foundation's community guidelines.
+- Consistent track record holding events (virtual or in person).
+- Meetup is listed with the .NET Foundation
+- Responsibility for tasks related to the meetup or conference.
+- Building a community among related groups or conferences. This could be sharing speakers, creating partnered events, or other activities that grow the community.
+
+## Educator
+
+See [Educator particpation type](membership-participation-types.md#the-educator).
+
+We look to the educator's creation and delivery of curriculum in any of the following ways:
+
+- Teaching bootcamp courses, college or high school level courses in software development, where the .NET platform is a signification part of the curriculum.
+- Creating .NET based curriculum for any of these area.
+
+## Knowledge sharers
+
+See [Sharer of knowledge particpation type](membership-participation-types.md#the-sharer-of-knowledge).
+
+We look at the content they create and deliver to spread knowledge. These include:
```

Comment:
The content that the applicant either creates and/or delivers to spread knowledge. These include, but are not limited to: